src/main/java/com/williamfiset/algorithms/datastructures/priorityqueue/BinaryHeap.java

```diff
@@ -130,7 +118,7 @@ private void swim(int k) {
   }
 
   // Top down node sink, O(log(n))
-  private void sink(int k) {
+  private void sink(int k, int heapSize) {
```

Comment:
Do we need to add the heapSize as a parameter? Can we capture the heapSize inside the method instead?

Reply:
Yes, you are right. Let me clean this.